### PR TITLE
Adding support for setting up a layout of the new tag.

### PR DESCRIPTION
**Pull request by geripgeri**
Example code

```awful.key({ modkey, "Shift" }, "n", function() lain.util.add_tag(awful.layout.suit.tile) end), -- add new tag```

Or by using: ```awful.layout.layouts[0]```

### CODE REVIEW

File: util/init.lua [hunk 1]
```diff
@@ -111,13 +111,13 @@ end
...
 -- Add a new tag
-function util.add_tag()
+function util.add_tag(layout)
     awful.prompt.run {
         prompt       = "New tag name: ",
         textbox      = awful.screen.focused().mypromptbox.widget,
         exe_callback = function(name)
             if not name or #name == 0 then return end
-            awful.tag.add(name, { screen = awful.screen.focused() }):view_only()
+            awful.tag.add(name, { screen = awful.screen.focused(), layout = layout }):view_only()
```
Comment (copycat-killer) [inline util/init.lua]: You should add `layout = layout or awful.layout.layouts[0]` to catch `nil` inputs.

Plus, a hint in the wiki too.

You can `git commit` the addition,  `git rebase -i HEAD~2` to squash it and `git push --force` to update this PR.

**Comment by geripgeri** [comment]
Thanks, I updated it.

File: util/init.lua (hunk 1 above)
Comment (geripgeri) [inline util/init.lua]: Thanks, I updated it.

**Comment by copycat-killer** [comment]
OK, I updated the wiki myself.

**Pull request merged**